x/did/client/cli/query_diddoc_version.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cheqd/cheqd-node/x/did/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+func CmdGetDidDocVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "did-version [id] [version-id]",
```

Comment:
Could this be a flag under the normal query command? E.g., `did-document [id] --version-id [version-id]`, where the flag is optional

Reply:
I suggest doing it later. It doesn't require any migrations. Would you agree?